src/components/common/animation-control/floating-time-display.tsx

```diff
@@ -24,7 +24,7 @@ import classnames from 'classnames';
 import {Minus} from 'components/common/icons';
 import {DEFAULT_TIME_FORMAT} from '@kepler.gl/constants';
 import {CenterFlexbox} from 'components/common/styled-components';
-import {datetimeFormatter} from 'utils/data-utils';
+import {datetimeFormatter} from '@kepler.gl/layers';
```

Comment:
why is this moved to `layers`? I don't see any layer using it. it is heavily used by components